runelite-client/src/main/java/net/runelite/client/plugins/itemcharges/ItemChargePlugin.java

```diff
@@ -75,10 +75,17 @@
 	private static final Pattern BINDING_USED_PATTERN = Pattern.compile(
 		"You bind the temple's power into (mud|lava|steam|dust|smoke|mist) runes\\.");
 	private static final String BINDING_BREAK_TEXT = "Your Binding necklace has disintegrated.";
+	private static final Pattern RING_OF_FORGING_CHECK_PATTERN = Pattern.compile(
+			"You can smelt ([0-9+]+|one) more pieces of iron ore before a ring melts\\.");
```

Comment:
Same indent issue here (and on the other added static vars), only increase the indentation level by one.
```suggestion
		"You can smelt ([0-9+]+|one) more pieces of iron ore before a ring melts\\.");
```